content/tutorials/get-started/python-env-setup.md

```diff
@@ -0,0 +1,116 @@
+---
+html: set-up-python-env.html
+funnel: Build
+doc_type: Tutorials
+category: Get Started
+subcategory: Get Started Using Python
+#parent: get-started-python.html - field not used by the site yet
+blurb: Set up your environment to start Python development.
+cta_text: Build Apps
+---
+
+## Python Environment Setup
+
+Complete the steps in the following sections to prepare your environment for development. 
```

Comment:
The steps described in this page are all helpful for developing/contributing to the xrpl-py library itself, but are not at all required for merely _using_ the library. In my opinion, instructions for developing the library belong in the library's repo and it's not necessary to duplicate them on xrpl.org.

I'm reasonably confident that the typical install will be as simple as running `pip3 install xrpl-py` (as long as you have Python 3.7 or newer) so we won't even need a full separate page for the installation/setup steps, and we can remove this entire page from here. It may be useful to contribute these changes back to the repo itself as a contributing guide though.